ptvsd/wrapper.py

```diff
@@ -613,7 +613,21 @@ def __init__(self, socket, pydevd, logfile=None, killonclose=True):
         self.event_loop_thread.daemon = True
         self.event_loop_thread.start()
 
+    def __input(self, message):
+        try:
+            raw_input(message)
```

Comment:
This will only work in Python 3.x, and even then it won't do what the message says ("press *any* key to continue"). It's not always easily doable, depending on the platform, but we can do a best effort. Here's the old code that does it - let's just take that:

https://github.com/Microsoft/PTVS/blob/master/Python/Product/PythonTools/ptvsd/debugger.py#L2535